src/helper/constants.ts

```diff
@@ -0,0 +1,37 @@
+//  A data about probable errors might need it later
+
+export const STANDARD = {
```

Comment:
Let's avoid having helper folder as much as we can. Because the issue is helper folders don't specify the purpose. It kinda contains a lot of stuff that could be mixed up.

Move all these to a file named something like`errors.ts` in root or even better would be moving server needs to separate thing